src/openrct2/park/ParkFile.cpp

```diff
@@ -1071,6 +1075,31 @@ namespace OpenRCT2
             }
         }
 
+        void UpdateBrakesClosedState()
+        {
+            for (int32_t y = 0; y < MAXIMUM_MAP_SIZE_TECHNICAL; y++)
```

Comment:
This doesn’t have to search the whole 1001 × 1001 tiles - use the actual map size here.